src/nyc_trees/js/src/event.js

```diff
@@ -2,11 +2,11 @@
 
 var $ = require('jquery'),
     L = require('leaflet'),
-    mapModule = require('./map'),
+    EventMap = require('./EventMap'),
     fetchAndReplace = require('./fetchAndReplace');
 
 var $map = $('#map');
-mapModule.create({
+new EventMap({
```

Comment:
I created a class so there would be a place to store the Leaflet `circleMarker` so it could be repositioned when you change the location.